src/App.tsx

```diff
@@ -3,9 +3,17 @@ import Home from "./pages/Home";
 import SignUp from "./pages/SignUp";
 import SignIn from "./pages/SignIn";
 import ForgotPassword from "./pages/ForgotPassword";
+import Theme from "theme";
+import styled from "styled-components";
 
 function App() {
   // theme provider here
+  <Theme>
+    <div>
+      <h2>Test</h2>
+    </div>
+  </Theme>
+  
   return (
     <Routes>
```

Comment:
```suggestion
    <ThemeProvider theme={theme}>
       <Routes>
```

Reply:
and you import `theme` from `./theme`
```
import { theme } from './theme';
```

Reply:
Thank you! I will try this fix tomorrow.

Reply:
I've made the suggested edits, but it isn't rendering for me when I test it, and there are still errors being listed. Any ideas what I can do to fix it?